vowpalwabbit/global_data.h

```diff
@@ -440,6 +440,7 @@ struct vw
   LEARNER::base_learner* cost_sensitive;//a cost sensitive learning algorithm.
 
   void learn(example*);
+  void learn(multi_ex&);
```

Comment:
why & and not *? seems inconsistent with example*

Reply:
Personally, I'd prefer & since it avoids allowing operations like ++ which don't make sense.  I agree this is inconsistent though.  How viable is it to change learn(example*)?

Reply:
Done